package.json

```diff
@@ -34,6 +34,11 @@
     "watch-build": "./node_modules/.bin/nx watch --all -- pnpm -w nx build \"\\$NX_PROJECT_NAME\""
   },
   "dependencies": {
+    "@1password/op-js": "^0.1.13",
```

Comment:
why deps vs dev deps? probably doesn't matter at the root but seems inconsistent